content/en/_common/functions/js/options.md

```diff
@@ -54,17 +54,17 @@ shims
   ```
 
 target
-: (`string`) The language target. One of: `es5`, `es2015`, `es2016`, `es2017`, `es2018`, `es2019`, `es2020`, `es2021`, `es2022`, `es2023`, `es2024`, or `esnext`. Default is `esnext`.
+: (`string`) The language target. One of: `es5`, `es2015`, `es2016`, `es2017`, `es2018`, `es2019`, `es2020`, `es2021`, `es2022`, `es2023`, `es2024`, or `esnext`. The default is `esnext`.
```

Comment:
```suggestion
: (`string`) The language target. One of: `es5`, `es2015`, `es2016`, `es2017`, `es2018`, `es2019`, `es2020`, `es2021`, `es2022`, `es2023`, `es2024`, or `esnext`. Default is `esnext`.
```

Reply:
For right or wrong, "Default is something" is standard throughout the documentation. If we change this at some point, we need to change it everywhere. But please do not submit a PR for that now... there's a very large change set in progress and I don't want to deal with a hundred merge conflicts.